src/Domain/Donation.php

```diff
@@ -33,11 +35,7 @@ class Donation extends SalesforceWriteProxy
     public const MAXIMUM_CARD_DONATION = 25_000;
 
     public const MAXIMUM_CUSTOMER_BALANCE_DONATION = 200_000;
-    /**
-     * @var int
-     * @see Donation::$currencyCode
-     */
-    private int $minimumAmount = 1;
```

Comment:
For some reason Psalm seemed to think this property was unused. I think it might be a Psalm bug but I replaced it with a constant anyway. It can't really be a readonly property as readonly properties aren't allowed to have values set on declaration (since doing that would make them just like constants)